config/initializers/geocoder.rb

```diff
@@ -1,13 +1,14 @@
 Geocoder.configure(
   # Geocoding options
   # timeout: 3,                 # geocoding service timeout (secs)
-  # lookup: :nominatim,         # name of geocoding service (symbol)
+  lookup: :mapbox, # name of geocoding service (symbol)
+  country: 'sg',
   # ip_lookup: :ipinfo_io,      # name of IP address geocoding service (symbol)
   # language: :en,              # ISO-639 language code
   # use_https: false,           # use HTTPS for lookup requests? (if supported)
   # http_proxy: nil,            # HTTP proxy server (user:pass@host:port)
   # https_proxy: nil,           # HTTPS proxy server (user:pass@host:port)
-  # api_key: nil,               # API key for geocoding service
+  api_key: "pk.eyJ1IjoidGVld2h5a2F5IiwiYSI6ImNrc3lkZXB5azJpbWoybmt0MXQwMGVib3UifQ.1elDNvnLbuHZ1dm8NOu8fg", # API key for mapbox
```

Comment:
👀  This not like those that need to put in ENV?

Reply:
nope, geocoder gem will use the api_key internally to call mapbox api 😎🆒